graders/r/Dockerfile

```diff
@@ -31,49 +31,11 @@ ENV PYTHONIOENCODING=UTF-8
 # tidyverse'. Switching to using the CRAN package names may make it easier to other R
 # users to adapt and adjust the package list.  We also install sqlite3 to support the
 # SQL lectures. (Adding a duckdb binary may be an option too.)
-RUN apt update \
-    && apt install -y curl \
-                      git \
-                      pandoc \
-                      sqlite3 \
-                      sudo \
-    && install.r bench \        
-                 data.table \
-                 devtools \
-                 diffobj \
-                 doparallel \
-                 duckdb \
-                 dygraphs \
-                 flexdashboard \
-                 foreach \
-                 fs \
-                 future.apply \
-                 gapminder \
-                 gh \
-                 git2r \
-                 igraph \
-                 lahman \
-                 lintr \
-                 magick \
-                 memoise \
-                 microbenchmark \
-                 nycflights13 \
-                 palmerpenguins \
-                 png \
-                 profmem \
-                 RcppArmadillo \
-                 rex \
-                 rsqlite \
-                 runit \
-                 shiny \
-                 stringdist \
-                 testthat \
-                 tidyverse \
-                 tinysnapshot \
-                 tinytest \
-                 ttdo \
-                 unix \
-                 xts \
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends curl git pandoc sqlite3 sudo \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* \
+    && install.r bench data.table devtools diffobj doparallel duckdb dygraphs flexdashboard foreach fs futu apply gapminder gh git2r igraph lahman lintr magick memoise microbenchmark nycflights13 palmerpenguins png profmem RcppArmadillo rex rsqlite runit shiny stringdist testthat tidyverse tinysnapshot tinytest ttdo unix xts \
```

Comment:
IMO this is actually a formatting regression. There are sufficiently many packages here that wrapping made it much easier to read.